.github/workflows/build.yml

```diff
@@ -282,7 +282,7 @@ jobs:
     - name: Verify Python bindings
       run: |
         export LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}"
-        export PYTHON_VERSION_MINOR=$(_tmp=$(find /usr/local/lib/ -type d -iname "python*") && echo ${_tmp: -3}; unset _tmp)
+        export PYTHON_VERSION_MINOR=$(_tmp=$(find /usr/local/lib/ -maxdepth 1 -type d -iname "python3.*") && echo ${_tmp: -3}; unset _tmp)
```

Comment:
This fixes a depth of find bug on the runners that was exposed when the runner environments updated.

Reply:
erm, for minor of python version why not `python3 -c 'import sys; print(sys.version_info.minor)'` ?

Reply:
Yeah
```yaml
export PYTHON_VERSION_MINOR=$(python3 -c 'import sys; print(f"python{sys.version_info.major}.{sys.version_info.minor}")')
```

is more stable then using `find`

Reply:
Applied in the rebase :+1: 

```diff
-        export PYTHON_VERSION_MINOR=$(_tmp=$(find /usr/local/lib/ -type d -iname "python*") && echo ${_tmp: -3}; unset _tmp)
-        export PYTHONPATH="/usr/local/lib/python${PYTHON_VERSION_MINOR}/site-packages:${PYTHONPATH}"
+        export PYTHON_VERSION_MINOR=$(python3 -c 'import sys; print(f"{sys.version_info.minor}")')
+        export PYTHONPATH="/usr/local/lib/python3.${PYTHON_VERSION_MINOR}/site-packages:${PYTHONPATH}"
```